### ISSUE TITLE
use Enum for left/right/center and auto/never/always

### ISSUE THREAD

**Issue by igor-raits** [ignatenkobrain/python-smartcols#5]
Currently we do `cdef dict ....` and then query in it.

But this way we can compare only strings and not objects..

``` python
>>> import smartcols
>>> tb = smartcols.Table()
>>> tb.termforce
'auto'
```

**Comment by igor-raits** [comment]
@jsilhan thoughts?

**Comment by igor-raits** [comment]
Fixed in dfd9b37
